sky/serve/autoscalers.py

```diff
@@ -183,6 +184,7 @@ def _get_desired_num_replicas(self) -> int:
         # is not enabled, i.e. self.target_qps_per_replica is None.
         # In this case, self.target_num_replicas will be min_replicas.
         if self.target_qps_per_replica is None:
+            self.bootstrap_done = True
```

Comment:
This seems having no effect, as this if condition will always short circuit the following calculation?
If it is the case, might be good to have a comment.

Reply:
This matters earlier in a subtle case where we do not set `self.bootstrap_done=False` during update. 
I think since we decide to do that, as you mentioned, this will not have effect.